scripts/docker/gatkbase/Dockerfile

```diff
@@ -15,23 +15,28 @@ RUN apt-get update && \
     less \
     bedtools \
     samtools \
-    openjdk-8-jdk \
     tabix \
     gpg-agent \
     build-essential \
+    openjdk-17-jdk \
     software-properties-common && \
+	
```

Comment:
Is this empty line not deliberate ? Does it not matter ?

Reply:
huh, that's definitely not deliberate and I would have expected it to make it break...

Reply:
It seems to make no difference.